src/EncompassRest.Tests/LoanConditionsTests.cs

```diff
@@ -21,10 +21,11 @@ public async Task LoanConditions_GetUnderwritingConditions()
                 var conditions = await underwritingConditions.GetConditionsAsync();
                 Assert.IsNotNull(conditions);
                 Assert.AreEqual(0, conditions.Count);
-                var addedCondition = new UnderwritingCondition { Title = "ABC", Source = ConditionSource.RecordersOffice, ForAllApplications = true };
+                var addedCondition = new UnderwritingCondition { Title = "ABC", Source = ConditionSource.RecordersOffice, ForAllApplications = true, OwnerRole = new EntityReference("6", EntityType.Role) };
                 await underwritingConditions.CreateConditionsAsync(new[] { addedCondition });
                 var conditionId = addedCondition.Id;
                 Assert.IsFalse(string.IsNullOrEmpty(conditionId));
+                Assert.AreEqual(addedCondition.OwnerRole.EntityId, "6");
```

Comment:
expected value of "6" should be the first argument, the actual value of `addedCondition.OwnerRole.EntityId` should be the second argument.